### PR TITLE
make LF 1.16 stable

### PR DESCRIPTION
**Pull request by remyhaemmerle-da**

### CODE REVIEW

File: sdk/daml-lf/transaction/src/test/scala/com/digitalasset/daml/lf/transaction/TransactionSpec.scala [hunk 1]
```diff
@@ -291,7 +297,6 @@ class TransactionSpec
       val parties = Set("Alice")
 
       def create(s: V.ContractId) = {
-        println(s)
```
Comment (rgugliel-da) [inline sdk/daml-lf/transaction/src/test/scala/com/digitalasset/daml/lf/transaction/TransactionSpec.scala:294]: :)

File: sdk/daml-lf/transaction-test-lib/src/main/scala/lf/transaction/test/TestNodeBuilder.scala [hunk 2]
```diff
@@ -72,7 +74,7 @@ trait TestNodeBuilder {
 
     Node.Create(
       coid = id,
-      packageName = packageName.filter(_ => transactionVersion < TransactionVersion.minUpgrade),
+      packageName = packageName.filter(_ => transactionVersion >= TransactionVersion.minUpgrade),
```
Comment (dylant-da) [inline sdk/daml-lf/transaction-test-lib/src/main/scala/lf/transaction/test/TestNodeBuilder.scala:77]: Why did this comparator change to the exact opposite? Was there a bug in the original line?

File: sdk/daml-lf/language/daml-lf.bzl [hunk 3]
```diff
@@ -75,7 +75,7 @@ lf_version_latest = {
 lf_version_configuration = {
     "default": "1.15",
```
[the remark this answers is not in the document]
Comment (remyhaemmerle-da) [reply sdk/daml-lf/language/daml-lf.bzl:76]: 1.16 will be default in the next SDK, i.e. upgrade is opt in for SDK 2.9.

File: sdk/daml-lf/language/daml-lf.bzl [hunk 4]
```diff
@@ -75,7 +75,7 @@ lf_version_latest = {
...
     "latest": lf_version_latest.get("1"),
-    "preview": "1.16",
+    #    "preview": "1.16",
```
[the remark this answers is not in the document]
Comment (dylant-da) [reply sdk/daml-lf/language/daml-lf.bzl:78]: Agree

File: sdk/daml-lf/transaction-test-lib/src/main/scala/lf/transaction/test/TestNodeBuilder.scala (hunk 2 above)
Comment (remyhaemmerle-da) [reply sdk/daml-lf/transaction-test-lib/src/main/scala/lf/transaction/test/TestNodeBuilder.scala:77]: It looks like a bug in the original code -- maybe a confusion between `filter` and `filterNot`. 
packageName should be define when `transactionVersion >= TransactionVersion.minUpgrade`.
cc @simonmaxen-da

**Pull request merged**